src/main/java/com/google/jenkins/plugins/credentials/oauth/GoogleRobotCredentials.java

```diff
@@ -64,35 +67,41 @@ protected GoogleRobotCredentials(String projectId, GoogleRobotCredentialsModule
    *     be either GLOBAL or SYSTEM.
    * @param id The credential ID to assign.
    * @param projectId The project id with which this credential is associated.
+   * @param description The credential description
    * @param module The module to use for instantiating the dependencies of credentials.
    */
   protected GoogleRobotCredentials(
       @CheckForNull CredentialsScope scope,
       String id,
       String projectId,
+      String description,
       GoogleRobotCredentialsModule module) {
-    super(scope, id == null ? "" : id, Messages.GoogleRobotCredentials_Description());
+    super(scope, id == null ? "" : id,
+          description == null || description.isEmpty()
```

Comment:
I prefer to remove that logic. I didn't see something similar in other types of credentials.

Reply:
Agree on that. Previously, we didn't specify the description, but now we can specify it through UI.
Therefore, it's better not to set a default message if the user prefers to leave the credential description empty